docs-partials/builder/vsphere/clone/WindowsOptions-not-required.mdx

```diff
@@ -0,0 +1,33 @@
+<!-- Code generated from the comments of the WindowsOptions struct in builder/vsphere/clone/step_customize.go; DO NOT EDIT MANUALLY -->
+
+- `run_once_command_list` (\*[]string) - CustomizationGuiRunOnce
+  A list of commands to run at first user logon, after guest customization.
+
+- `auto_logon` (\*bool) - CustomizationGuiUnattended
+  Specifies whether or not the VM automatically logs on as Administrator.
+
+- `auto_logon_count` (\*int32) - Specifies how many times the VM should auto-logon the Administrator account when auto_logon is true. Default 1
+
+- `admin_password` (\*string) - The new administrator password for this virtual machine.
+
+- `time_zone` (\*int32) - The new time zone for the virtual machine. This is a sysprep-dictated timezone code. Default 85 (GMT)
+
+- `domain_admin_user` (string) - CustomizationIdentification
+  The user account of the domain administrator used to join this virtual machine to the domain.
+
+- `domain_admin_password` (string) - The password of the domain administrator used to join this virtual machine to the domain.
+
+- `join_domain` (string) - The domain that the virtual machine should join.
```

Comment:
I don't think it makes good sense to include the ability to join the machine image to a domain per comments in #34.

**Ryan Johnson**
Staff II Solutions Architect | VMware, Inc.